Add russian translation for the intermediate-ziplines.

alnero opened the pull request.

bugron commented:
@alnero привет.
**Добавьте стиль историям из Camper News**
***1.*** Вы везде перевели `Build a CodePen.io app` как `Создайте CodePen.io`. Отсутствует слово **приложение** после 'Создайте'. Понимаете, `Создайте CodePen.io` читатель (особенно новичок) поймет как 'Создайте **сайт** CodePen.io' (но ведь сайт CodePen.io уже существует), поэтому нужно писать 'Создайте **приложение** CodePen.io'. Только что заметил, что то же самое относится и к **basic-ziplines.json**. Следуйте инструкциям из [моего комментария](https://github.com/FreeCodeCamp/FreeCodeCamp/pull/3351#issuecomment-141769461), только вместо `git commit -m "add russian translation for the basic-ziplines"` сделайте `git commit -m "add russian translation for the intermediate-ziplines and fix basic-ziplines"`. Ну и в данном случае, естественно, подставляете соответствующие названия. Извиняюсь за это неудобство.
***2.*** Честно говоря 'API свежих историй с Camper News' звучит не по сути. Там нету никаких API, есть только ответ на простой GET запрос в формате JSON. По моему в этом случае лучше отойти от оригинала и написать 'Ссылка на свежие истории с Camper News в формате JSON'. Но по данному вопросу пока еще думаю, что скажете?

Перевод **Поиск в Википедии** отличный.

**Создайте калькулятор на JavaScript**
'Я могу друг за другом вводить математические операции до тех пор, пока не нажму кнопку сброса, после этого калькулятор должен показать мне правильный результат вычислений.'
Думаю должно быть так:
'Я могу друг за другом вводить математические операции и если я не нажму на кнопку сброса, то калькулятор должен показать мне правильный результат вычислений.'

**Создайте игру крестики-нолики**
'В качестве пользователя, я могу играть в крестики нолики с компьютером.' => 'крестики нолики' должно быть 'крестики-нолики'.

Пока, наверное, все. Завтра посмотрю **Создайте игру Саймона**.

alnero committed:
Add russian translation for the intermediate-ziplines and fix basic-ziplines

ahstro commented:
I think it'd be better if you used English in the comment sections so the rest of us can, at least somewhat, follow the discussion.

alnero commented:
Sure we can use English, but it will somehow look weird when you are explained in English what is wrongly translated in Russian when you both are native speakers)

Re the last edit:

I've fixed missing translation of CodePen.io *app* in both basic and intermediate bonfires. Changed some word endings as well.

"API свежих историй с Camper News" sounds terribly, so I've changed to your version.

In JS Calculator also made translation close to your version, pls check.

Tic Tac Toe missing dash added.

bugron commented:
@alnero, вот так лучше, продолжаем.
**Создайте игру Саймона** => это некорректный перевод. **Саймон** - это название игры, а не человек, который, допустим, создал эту игру. Правильнее будет **Создайте игру Саймон** или **Создайте игру Simon**.
"В качестве пользователя, если я захочу сделать рестарт, то могу для этого нажать на кнопку, и игра начнется с первоначального шага." => оригинал имеет ввиду не первоначальный шаг (который читатель поймет как "первоначальный шаг текущей **серии нажатий**"), а просто новую игру с первого шага и, на самом деле, первоначальный шаг этой новой игры отличается от первоначального шага предыдущей игры (серии нажатий). Именно так себя ведет игра из примера. Допустим, если игра началась с желтого, то после того, как я нажимаю на кнопку Start начинается новая игра, но уже с другого цвета (если новая игра начнется с желтого, то это просто совпадение).

Пока все. Надеюсь в следуюший раз. после проверки новых изменеий, сделать `merge`. И еще один момент. Я по глупости сказал, что можно не дожидаться слияния текущего PR и открыть новый. Но опыт с **приложение**м показал, что лучше сделть все последовательно, а не параллельно. Просто так Вам будет проще исправлять что-то из предыдущих PRов, потому что перед началом работы в новой ветке Вы делаете `git fetch` и `git merge`,  то есть перед началом работы над новым файлом у Вас уже есть все изменения предыдущего и того Вам не придеться выполнять те же самые дествия из моего первого комментария.

alnero commented:
Варианты:
- В качестве пользователя, если я захочу сделать рестарт, то могу для этого нажать на кнопку, и новая игра начнется с первого шага.
- В качестве пользователя, если я захочу начать новую игру, то могу для этого нажать на кнопку, и игра начнется с первого шага.
- В качестве пользователя, если захочу я могу, нажав на кнопку, начать новую игру с первого шага.

Мне нравится последний. Какой лучше?

bugron commented:
@alnero мне тоже третий нравится, хотя первый кажется ближе к оригиналу.
По поводу будущих PRов. Кажется намечаются изменения в 'zipline'ах, разработчики хотят то ли удалить/добавить новые, то ли изменить порядок, какие-то перенести. После того, как закончим работу над текущим PRом, лучше взять 'таймаут' на пару недель и продолжить после их изменений. Я буду держать в курсе последних событий.

Klu6ok commented:
всем привет. Когда планируется ввести первый тест на русском языке ?)

QuincyLarson commented:
Hi @alnero and @bugron - I apologize that it has taken me so long to merge this in. I can't read Russian so I can't easily address these merge conflicts. Could you submit a new pull request that adds the Russian translations to the corresponding challenges (which have since moved around - for example, all the API-centric Ziplines are now in "intermediate ziplines". Thanks for your help with this.
